### PR TITLE
Refactoring from css to scss

### PR DESCRIPTION
**Pull request by blackinno**
- Create a theme.
  - variables
- Add a theme to each component
- Storybook
  - config webpack to use scss library
  - fixed components at storybook to use scss

Closed. https://github.com/mesg-foundation/mesg-components/issues/41

### CODE REVIEW

**Comment by EmmanuelDrouin** [comment]
Already fixed

File: packages/Theme/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+$primary-very-light: #bdafd6 !default;
+$primary-very-dark: #210e41 !default;
+$dark-grey: #3c3940 !default;
+$light-grey: #f2f0f7 !default;
+$White: #ffffff !default;
+$Black: #000000 !default;
+
+$Error: #f44336 !default;
+$Success: #4bb543 !default;
+$Info: #1e88e5 !default;
```
Comment (antho1404) [inline packages/Theme/_variables.scss]: Could you make these variables lowercase to have some consistency

**Comment by antho1404** [comment]
Pull request reopened because of a confusion with https://github.com/mesg-foundation/mesg-components/pull/54.

Let's finalize and merge this one then update all the branches and pull requests for the new components.

File: packages/Theme/_variables.scss (hunk 1 above)
Comment (blackinno) [reply packages/Theme/_variables.scss]: Fixed

File: packages/Button/Button.vue [hunk 2]
```diff
@@ -88,8 +90,8 @@ i {
 }
 
 .btn--primary {
-  color: var(--White);
-  background-color: var(--primary);
+  color: $white;
+  background-color: $primary;
 }
 .btn--primary:hover {
   background-color: var(--secondary);
```
Comment (antho1404) [inline packages/Button/Button.vue]: You shouldn't have any `var()` nowhere in the codebase, that's the whole purpose of moving to `scss`. I've seen a few in many files that were not here before the resolution of conflicts.

I don't think the resolution of conflicts was done correctly...

**Review by antho1404 — CHANGES_REQUESTED** [review]

File: packages/Button/Button.vue (hunk 2 above)
Comment (blackinno) [reply packages/Button/Button.vue]: Fixed

**Review by antho1404 — APPROVED** [review]
I don't think the `Theme/index.scss` and `Theme/_structure.scss` should be in the component (at least yet) and at best should be moved to the storybook for now.

But let's do that later if it's needed.

@EmmanuelDrouin, if you don't see any issue in the components, it's good to merge for me.

**Review by EmmanuelDrouin — APPROVED** [review]
This PR is good for me. @antho1404 let's Merge this one and reopen the others PR so @blackinno can implement his previous work on the components again. On this one all the work he did previously on the design of the components and the creation of the missing components disappear.

**Pull request merged**